snuba/datasets/entities/discover.py

```diff
@@ -446,7 +446,7 @@ def __init__(self) -> None:
         events_ro_storage = get_storage(StorageKey.EVENTS_RO)
         transactions_storage = get_storage(StorageKey.TRANSACTIONS)
 
-        self.__time_group_columns: Mapping[str, str] = {}
+        self.__time_group_columns: Mapping[str, str] = {"time": "timestamp"}
         self.__time_parse_columns = ("timestamp",)
```

Comment:
You do not need these two fields anymore. Could you just pass the dictionary and the tuple directly to the initializer of the TimeSeriesProcessor ?